packages/commonwealth/client/scripts/views/pages/Editor/Editor.tsx

```diff
@@ -0,0 +1,262 @@
+    },
+    [handleDropAsync],
+  );
+
+  const handleDragEnter = useCallback((event: React.DragEvent) => {
+    console.log('handleDragStart');
+    event.preventDefault();
+    dragCounterRef.current = dragCounterRef.current + 1;
+
+    if (dragCounterRef.current === 1) {
+      setDragging(true);
+    }
+  }, []);
+
+  const handleDragOver = useCallback((event: React.DragEvent) => {
+    console.log('handleDragOver');
+
+    event.preventDefault();
+    // This is necessary to allow a drop
+    event.dataTransfer!.dropEffect = 'copy'; // Shows a copy cursor when dragging files
+  }, []);
+
+  const handleDragLeave = useCallback((event: React.DragEvent) => {
+    console.log('handleDragLeave');
+    event.preventDefault();
+    dragCounterRef.current = dragCounterRef.current - 1;
+
+    if (dragCounterRef.current === 0) {
+      setDragging(false);
+    }
+  }, []);
+
+  return (
+    <div
+      className={clsx(
+        'mdxeditor-container',
+        'mdxeditor-container-mode-' + mode,
+      )}
+      onDrop={handleDrop}
+      onDragEnter={handleDragEnter}
+      onDragOver={handleDragOver}
+      onDragLeave={handleDragLeave}
+    >
+      <MDXEditor
+        onError={errorHandler}
+        ref={mdxEditorRef}
+        markdown={supported}
+        placeholder={placeholder}
+        iconComponentFor={iconComponentFor}
+        translation={(key, defaultValue, interpolations) => {
+          switch (key) {
+            case 'toolbar.blockTypeSelect.placeholder':
+              // show the default placeholder that's active here..
+              return 'H1';
+            case 'toolbar.blockTypes.heading':
+              if (interpolations?.level) {
+                // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+                return 'H' + interpolations.level;
+              }
+              return 'H1';
+            case 'toolbar.blockTypes.quote':
+              return 'Q';
+            case 'toolbar.blockTypes.paragraph':
+              return 'P';
+            default:
+              return defaultValue;
+          }
+        }}
+        plugins={[
+          toolbarPlugin({
+            location: mode === 'mobile' ? 'bottom' : 'top',
+            toolbarContents: () =>
+              mode === 'mobile' ? <ToolbarForMobile /> : <ToolbarForDesktop />,
+          }),
+          listsPlugin(),
+          quotePlugin(),
+          headingsPlugin(),
+          linkPlugin(),
+          linkDialogPlugin(),
+          codeBlockPlugin({ defaultCodeBlockLanguage: 'js' }),
+          codeMirrorPlugin({
+            codeBlockLanguages,
+          }),
+          imagePlugin({ imageUploadHandler }),
+          tablePlugin(),
+          thematicBreakPlugin(),
+          frontmatterPlugin(),
+          diffSourcePlugin({ viewMode: 'rich-text', diffMarkdown: 'boo' }),
+          markdownShortcutPlugin(),
+        ]}
+      />
+
+      {mode === 'desktop' && (
+        <DesktopEditorFooter onImportMarkdown={handleImportMarkdown} />
```

Comment:
Bug: Importing a markdown file works the first time, but doesn't work afterwards - have to reload the page to make it work

Reply:
Can you explain how you got it to fail? I can't ... it works for me to import markdown twice.

Reply:
like this


https://github.com/user-attachments/assets/93a19d00-690c-49ed-af31-801058d2ee10

Reply:
I think that you mean that the drop indicator doesn't work the second time? I've confirmed that and not sure why that's happening yet.

Reply:
OK.  Fixed it. It happened if you imported the same filename twice!